validator/src/bootstrap.rs

```diff
@@ -653,6 +653,15 @@ fn get_rpc_nodes(
             cluster_info,
             &rpc_peers,
             validator_config.known_validators.as_ref(),
+            if newer_cluster_snapshot_timeout
+                .as_ref()
+                .map(|timer: &Instant| timer.elapsed().as_secs())
+                < Some(60)
```

Comment:
it's somewhat unintuitive that `All` is the default due to `None < Some(_)` being true. consider switching to `.map(|t| t.elapsed() < Duration::from_secs(60)).unwrap_or(true)`. also pop `Duration::from_secs(60)` out to a proper named `const`